src/content/pages/faq.mdx

```diff
@@ -5,11 +5,37 @@ subtitle: Frequently Asked Questions about EuroPython
 
 # FAQ
 
+## **Milestones**
+<ul className="milestone-done">
+<li><b>2024-01-27</b>: [Call for Contributors](/https://blog.europython.eu/seize-the-opportunity-europython-2024-call-for-contributors-extended/) Sign-up Starts </li>
+<li><b>2024-02-12</b>: [Call for Contributors](/https://blog.europython.eu/seize-the-opportunity-europython-2024-call-for-contributors-extended/) Sign-up Closes </li>
+<li><b>2024-02-14</b>: [Speaker Mentorship Programme](/mentorship) Sign-up for Mentees Starts </li>
+<li><b>2024-02-14</b>: [Call for Proposals (CFP)](/cfp) Starts </li>
+<li><b>2024-02-27</b>: [Speaker Mentorship Programme](/mentorship) Sign-up for Mentes Closes </li>
+<li><b>2024-03-08</b>: [Financial Aid Programme](/finaid) Starts </li>
+<li><b>2024-02-26</b>: [Speaker Mentorship Programme Ask Me Anything Event](/mentorship#ask-me-anything-about-cfp-workshop---tuesday-14-march-2023-1800-cet)</li>
+<li><b>2024-03-13</b>: Call for Proposals (CFP) Ends </li>
+</ul>
+
+<ul className="milestone-todo">
+<li><b>2024-03-25</b>: [Ticket Sales](/tickets) Start</li>
+<li><b>2024-04-21</b>: [Financial Aid Programme](/finaid) First Round Closes </li>
+<li><b>TBD</b>: [Sponsorship](/sponsor) Sign-up Starts </li>
+<li><b>TBD</b>: [Community Voting](/voting) starts </li>
+<li><b>TBD</b>: Community Voting ends </li>
+<li><b>2024-07-08</b>: Together at EuroPython 2024! 🎊 </li>
+
+</ul>
+
 ## **Q. When and where is EuroPython 2024 taking place?**
 
 A. EuroPython 2024 will be held on **8-14 July 2024**. Put the dates in your diary!
 
-The conference will be held in Prague, Czech Republic.
+The conference will be held at <b>[The Prague Congress Centre, the PCC](https://www.praguecc.cz/en/homepage)</b> in Prague, Czech Republic.
```

Comment:
```suggestion
The conference will be held at **[The Prague Congress Centre, the PCC](https://www.praguecc.cz/en/homepage)** in Prague, Czech Republic.
```
We could use the markdown style for bold. Maybe something else was broken and using `<b>` was solving it?